hrjobcontract/js/test/services/contract-health_test.js

```diff
@@ -50,11 +35,68 @@ define([
       });
 
       it('calls getOne fuction and return expected values', function () {
-        ContractHealthService.getOne({jobcontract_revision_id: 68}).then(function (result) {
+        ContractHealthService.getOne({
+          jobcontract_revision_id: 68
+        }).then(function(result) {
           expect(result.id).toEqual("47");
           expect(result.jobcontract_revision_id).toEqual(responseData.values[0].jobcontract_revision_id);
         });
       });
     });
+
+    describe('when calling getOptions()', function () {
+      it('defines getOptions() function', function () {
+        expect(ContractHealthService.getOptions).toBeDefined();
```

Comment:
this shouldn't be here, in this section you are testing for stuff that should happen when you call the method, you are not testing if the method is present or not.

If you want to test for that, then that should be among the first tests that you write
```js
describe('ContractHealthService', function() {

  it('has the expected api', function () {
    expect(ContractHealthService.getOptions).toBeDefined();
    // and so on with the other methods
  });
});